### PR TITLE
mobile version completed

### PR DESCRIPTION
**Pull request by Martha-K**

### CODE REVIEW

File: src/pages/Products.tsx [hunk 1]
```diff
@@ -1,49 +1,22 @@
...
-      </div>
+      {mockProduct?.map((el) => (
+        <div key={Object.keys(el)[0]}>
+          <h2>{(el?.armature || el?.grid)?.[0]?.name}</h2>
+          {(el?.armature || el?.grid).map((el) => (
+            <div key={el.name}>
+              <h3>{el.name}</h3>
+              <img src={el.image} alt="product" />
+              <p>Price: {el.price}</p>
+              {el.inStock && <p>В наявності</p>}
```
Comment (gennadiy123) [inline src/pages/Products.tsx:15]: add translation here please

File: src/components/Header/headerStyles.ts [hunk 2]
```diff
@@ -1,20 +1,60 @@
...
+  padding: 0;
+  margin: 0;
+  margin-top: 10px;
+
+  background: #2c3e50;
+  position: absolute;
+  top: 70px;
+  left: 0;
+  width: 100%;
+  z-index: 1000;
```
Comment (gennadiy123) [inline src/components/Header/headerStyles.ts:27]: do we need this?

File: src/components/Header/headerStyles.ts [hunk 3]
```diff
@@ -1,20 +1,60 @@
...
+export const MobileNavLink = styled(NavLink)`
+  color: white;
+  text-decoration: none;
+  font-size: 18px;
+`;
+
+export const HeaderWrapper = styled.div`
+  position: sticky;
+  top: 0;
+  z-index: 1000;
```
Comment (gennadiy123) [inline src/components/Header/headerStyles.ts]: maybe better to change it to 2?

File: src/components/Header/headerStyles.ts [hunk 4]
```diff
@@ -59,6 +102,20 @@ export const Li = styled.li`
...
 `;
 
+export const Burger = styled.div`
+  margin-right: 50px;
+  display: none;
+  background-image: url(${burger});
+  width: 40px;
+  height: 40px;
+  background-position: center;
+  background-size: 50px;
```
Comment (gennadiy123) [inline src/components/Header/headerStyles.ts:106]: add here cursor: pointer please

File: src/components/Header/headerStyles.ts (hunk 2 above)
Comment (Martha-K) [reply src/components/Header/headerStyles.ts:27]: yes, these are styles that control the position of the mobile menu

File: src/pages/Products.tsx (hunk 1 above)
Comment (Martha-K) [reply src/pages/Products.tsx:15]: Done

File: src/components/Header/headerStyles.ts (hunk 4 above)
Comment (Martha-K) [reply src/components/Header/headerStyles.ts:106]: I think it makes no sense to change the cursor for the menu of the mobile version

File: src/components/Header/headerStyles.ts (hunk 3 above)
Comment (Martha-K) [reply src/components/Header/headerStyles.ts]: I removed these styles altogether, they are not needed

**Comment by Martha-K** [comment]
Done

**Pull request merged**